docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: Database Access with Redshift on AWS
+description: How to configure Teleport Database Access with AWS Redshift PostgreSQL
+---
+
+# AWS Redshift PostgreSQL
+
+## Prerequisites
+
+Collect a few pieces of information about your Redshift cluster which you'll
+need to configure access.
+
+- Cluster identifier can be found on the `General information` panel:
+
+![Redshift Cluster ID](../../../img/database-access/guides/redshift/cluster-id@2x.png)
+
+- Cluster endpoint can also be copied from the `General information` panel:
+
+![Redshift Cluster Endpoint](../../../img/database-access/guides/redshift/cluster-endpoint@2x.png)
```

Comment:
Yeah it's pretty cool. I think we can use similar approach for things like configuring GUI database clients, instead of a bunch of screenshots. I'll keep this in mind next time I'm extending GUI clients section.